### DISCUSSION TITLE
What's the best way to use primary and secondary color?

### DISCUSSION THREAD

**Discussion by PiochU19** [react-page/react-page#1226] [Q&A]
Hi, as in the title. Generally I would like to control Editor by 'global' variables. I was thinking about css variables, but maybe there is some easier way? Thanks in advance :)

**Comment by macrozone** [comment]
you can pass in a custom theme, see a simple example here https://github.com/react-page/react-page/blob/master/examples/pages/examples/dark-editor.tsx

the theme your passing in is a mui theme, so best consult the docs there (or use the type definitions) how to set primary and secondary colors
